examples/discover_customization_examples/public/plugin.tsx

```diff
@@ -173,72 +173,6 @@ export class DiscoverCustomizationExamplesPlugin implements Plugin {
             </EuiFlexItem>
           );
         },
-      });
-
-      customizations.set({
-        id: 'search_bar',
-        CustomDataViewPicker: () => {
-          const [isPopoverOpen, setIsPopoverOpen] = useState(false);
-          const togglePopover = () => setIsPopoverOpen((open) => !open);
-          const closePopover = () => setIsPopoverOpen(false);
-          const [savedSearches, setSavedSearches] = useState<
-            Array<SimpleSavedObject<{ title: string }>>
-          >([]);
-
-          useEffect(() => {
-            core.savedObjects.client
-              .find<{ title: string }>({ type: 'search' })
-              .then((response) => {
-                setSavedSearches(response.savedObjects);
-              });
-          }, []);
-
-          const currentSavedSearch = useObservable(
-            stateContainer.savedSearchState.getCurrent$(),
-            stateContainer.savedSearchState.getState()
-          );
-
-          return (
-            <EuiFlexItem grow={false}>
-              <EuiPopover
-                button={
-                  <EuiButton
-                    iconType="arrowDown"
-                    iconSide="right"
-                    fullWidth
-                    onClick={togglePopover}
-                    data-test-subj="logsViewSelectorButton"
-                  >
-                    {currentSavedSearch.title ?? 'None selected'}
-                  </EuiButton>
-                }
-                isOpen={isPopoverOpen}
-                panelPaddingSize="none"
-                closePopover={closePopover}
-              >
-                <EuiContextMenu
-                  size="s"
-                  initialPanelId={0}
-                  panels={[
-                    {
-                      id: 0,
-                      title: 'Saved logs views',
-                      items: savedSearches.map((savedSearch) => ({
-                        name: savedSearch.get('title'),
-                        onClick: () => stateContainer.actions.onOpenSavedSearch(savedSearch.id),
-                        icon: savedSearch.id === currentSavedSearch.id ? 'check' : 'empty',
-                        'data-test-subj': `logsViewSelectorOption-${savedSearch.attributes.title.replace(
-                          /[^a-zA-Z0-9]/g,
-                          ''
-                        )}`,
-                      })),
-                    },
-                  ]}
-                />
-              </EuiPopover>
-            </EuiFlexItem>
-          );
-        },
```

Comment:
We somehow had a duplicate for `search_bar` customization. `CustomDataViewPicker` in the first one did not have the deprecated usage, the second one still did. 

<img width="412" height="239" alt="Screenshot 2025-09-10 at 14 50 40" src="https://github.com/user-attachments/assets/53b85008-3f58-42ab-94bf-86696e396f97" />


So I removed the duplicate and kept the `CustomDataViewPicker` which had the correct implementation.

<img width="388" height="129" alt="Screenshot 2025-09-10 at 14 53 17" src="https://github.com/user-attachments/assets/abfbb4f1-b3bf-454f-a084-edf93b48abd9" />